Functions.Templates/ProjectTemplate_v4.x/CSharp/Company.FunctionApp.csproj

```diff
@@ -5,7 +5,7 @@
     <RootNamespace Condition="'$(name)' != '$(name{-VALUE-FORMS-}safe_namespace)'">Company.FunctionApp</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="4.0.1" />

```

Comment:
@soninaren FYI. Looks like some of the recent changes in the v4.x branch was not moved to the restructured repo. for example, [this PR ](https://github.com/Azure/azure-functions-templates/pull/1171/files)had the version updating from 4.0.1 to 4.1.0, but it was not moved to the new structure in dev branch.